Sources/UIViewExtensions.swift

```diff
@@ -305,118 +312,118 @@ extension UIView {
 // MARK: Layer Extensions
 extension UIView {
     /// EZSwiftExtensions
-    public func setCornerRadius(radius radius: CGFloat) {
+    public func setCornerRadius(radius: CGFloat) {
         self.layer.cornerRadius = radius
         self.layer.masksToBounds = true
     }
     //TODO: add this to readme
     /// EZSwiftExtensions
-    public func addShadow(offset offset: CGSize, radius: CGFloat, color: UIColor, opacity: Float, cornerRadius: CGFloat? = nil) {
+    public func addShadow(offset: CGSize, radius: CGFloat, color: UIColor, opacity: Float, cornerRadius: CGFloat? = nil) {
         self.layer.shadowOffset = offset
         self.layer.shadowRadius = radius
         self.layer.shadowOpacity = opacity
-        self.layer.shadowColor = color.CGColor
+        self.layer.shadowColor = color.cgColor
         if let r = cornerRadius {
-            self.layer.shadowPath = UIBezierPath(roundedRect: bounds, cornerRadius: r).CGPath
+            self.layer.shadowPath = UIBezierPath(roundedRect: bounds, cornerRadius: r).cgPath
         }
     }
 
     /// EZSwiftExtensions
-    public func addBorder(width width: CGFloat, color: UIColor) {
+    public func addBorder(width: CGFloat, color: UIColor) {
         layer.borderWidth = width
-        layer.borderColor = color.CGColor
+        layer.borderColor = color.cgColor
         layer.masksToBounds = true
     }
 
     /// EZSwiftExtensions
-    public func addBorderTop(size size: CGFloat, color: UIColor) {
+    public func addBorderTop(size: CGFloat, color: UIColor) {
```

Comment:
`addBorder(width: CGFloat, color: UIColor, direction: [Direction])` something like this instead of many methods